libtestutils/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_policy(SET CMP0015 NEW)
+# this policy was introduced in cmake 3.0
+# remove if, once 3.0 will be used on unix
+if (${CMAKE_MAJOR_VERSION} GREATER 2)
+	# old policy do not use MACOSX_RPATH
+	cmake_policy(SET CMP0042 OLD)
+endif()
+set(CMAKE_AUTOMOC OFF)
+
+aux_source_directory(. SRC_LIST)
+
+include_directories(BEFORE ${JSONCPP_INCLUDE_DIRS})
+include_directories(BEFORE ..)
+include_directories(${MHD_INCLUDE_DIRS})
+include_directories(${JSON_RPC_CPP_INCLUDE_DIRS})
+include_directories(${LEVELDB_INCLUDE_DIRS})
+include_directories(${Boost_INCLUDE_DIRS})
+
+set(EXECUTABLE testutils)
+
+file(GLOB HEADERS "*.h")
+
+if (ETH_STATIC)
+	add_library(${EXECUTABLE} STATIC ${SRC_LIST} ${HEADERS})
+else()
+	add_library(${EXECUTABLE} SHARED ${SRC_LIST} ${HEADERS})
+endif()
+
+target_link_libraries(${EXECUTABLE} ${LEVELDB_LIBRARIES})
```

Comment:
Link dependencies seem wrong. Should be something like
libwebthree, libdevcore, libethereum, libdevcrypto, some json stuff and boost filesystem.

It should not depend on: leveldb, mhd, secp256k1, solidity, serpent.